src/forgot-password/messages.js

```diff
@@ -31,6 +31,11 @@ const messages = defineMessages({
     defaultMessage: 'Recover my password',
     description: 'Submit button text for the forgot password page.',
   },
+  'logisration.forgot.password.page.invalid.length.email.message': {
```

Comment:
semantics: I believe 'logisration.forgot.password.page.email.invalid.length.message' is more meaningful,

Reply:
done